monitoring/opentelemetry-reactive/src/test/java/io/quarkus/ts/opentelemetry/reactive/OpentelemetryReactiveIT.java

```diff
@@ -87,6 +89,35 @@ public void testSecurityEvents() {
 
     }
 
+    @Test
```

Comment:
I'd prefer if there was a tag with `QUARKUS-5424`.

Reply:
Add the tag